src/player_utils.c

```diff
@@ -1185,27 +1185,28 @@ TbBool player_sell_door_at_subtile(PlayerNumber plyr_idx, MapSubtlCoord stl_x, M
     {
         return false;
     }
-
-	struct Dungeon* dungeon = get_players_num_dungeon(thing->owner);
-	dungeon->camera_deviate_jump = 192;
-    long sell_value = compute_value_percentage(game.conf.doors_config[thing->model].selling_value, game.conf.rules.game.door_sale_percent);
-
-	dungeon->doors_sold++;
-	dungeon->manufacture_gold += sell_value;
-
+    struct DoorConfigStats *doorst = get_door_model_stats(thing->model);
+    struct Dungeon* dungeon = get_players_num_dungeon(thing->owner);
+    dungeon->camera_deviate_jump = 192;
+    GoldAmount sell_value = compute_value_percentage(doorst->selling_value, game.conf.rules.game.door_sale_percent);
+    dungeon->doors_sold++;
+    dungeon->manufacture_gold += sell_value;
     destroy_door(thing);
     if (is_my_player_number(plyr_idx))
-        play_non_3d_sample(115);
+    {
+        play_non_3d_sample(115); // TODO config make this sound configurable?
```

Comment:
I don't think it is a problem for the selling sound to be hardcoded. This is not a door specific thing.